nnvm/python/nnvm/frontend/keras.py

```diff
@@ -50,27 +50,45 @@ def _convert_activation(insym, keras_layer, _):
     elif act_type == 'softplus':
         return _sym.log(_sym.__add_scalar__(_sym.exp(insym), scalar=1))
     elif act_type == 'elu':
-        raise NotImplementedError('elu not implemented')
+        alpha = keras_layer.alpha if hasattr(keras_layer, "alpha") else 1
+        return -alpha * _sym.relu(1 - _sym.exp(insym)) + _sym.relu(insym)
+    elif act_type == 'selu':
+        alpha = keras_layer.alpha if hasattr(keras_layer, "alpha") else 1.6732
+        gamma = keras_layer.gamma if hasattr(keras_layer, "gamma") else 1.0507
```

Comment:
I think we should refer the original paper for these magic numbers; https://arxiv.org/abs/1706.02515.

Reply:
@kazum : these values are obtained from the same paper you shared.
Did you find any discrepancies?

Reply:
I mean we should add a comment to explain those values are from the paper.